brandon-brandonmcastillo/week-2/js-racer-game-lab/starter-code/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Race Game by Brandon Castillo</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Bangers" rel="stylesheet">
+</head>
+
+<body>
+    <!-- TITLE & MUSIC -->
+    <div>
+        <h1><img src="https://fontmeme.com/permalink/190201/e22a9f843e7c43e8e602bd1df92b5047.png" class="title" alt="fontmeme" border="0"></h1>
+        <input type="button" class="music" value="Click for fun" onclick="play()">
+        <audio id="song" src="Avengers.mp3"></audio>
+    </div>
+        <div>
+            <img src="infinity.png" class="gem">
+</div>
+            <!-- USE TABLES TO ADVANCE IMAGES  -->
+            <table class="gameArea">
+                <tr id="player1">
+                    <td class="marvel1"></td>
+                    <td> </td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+                <tr id="player2">
+                    <td class="marvel2"></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+            </table>
+            <div class="instructOne">
+                <p>Thanos: Press Z to warp</p>
+            </div>
+            <div class="instructTwo">
+                <p>Spiderman: Press -> to swing</p>
```

Comment:
Love it. Spiderman all the way.
that being said the structure of your code with all the <td> I would have liked to see you maybe implement this in a more dynamic way. For instance, using js to create the grid you needed.